src/app/clusters/account-login-server/codegen_sources.cmake

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2025 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is the equivalent to codegen_sources.gni
+TARGET_SOURCES(
+  ${APP_TARGET}
+  PRIVATE
+    "${CLUSTER_DIR}/account-login-delegate.h"
```

Comment:
account-login-delegate does not depend on specific app configuration.

I'm going to stop checking for these, because I expect this was just a mass-change that just assumed everything is dependent on codegen, but that's just not the case, and we should do a pass (followup to this PR is fine) that actually puts things in the right places just so we are clear on what work remains to decouple.

Reply:
That is correct: I codified in per-cluster config all the rules that we had before (i.e. add as monolity).
Decoupling would need to be done separately as there are over 130 clusters I believe.

Reply:
Keeping this open for note that updates should be as a followup.

I would also not directly followup here, however these would be done as we migrate things to "code driven clusters" as a sideffect.